### PR TITLE
Feat: add comments count & preview to story cards in feed

### PR DESCRIPTION
**Pull request by MTG2000**

### CODE REVIEW

**Comment by MTG2000** [comment]
@johnsBeharry 
Please review & if everything looks good, let's merge it.

**Comment by johnsBeharry** [comment]
quick thought. the comments button instead of being disabled if there's 0 it should bring you to the #comments section when clicked.

in another version lets try to resolve the npubs to nostr / bolt.fun profiles.

File: src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx [hunk 1]
```diff
@@ -87,7 +92,49 @@ export default function StoryCard({ story }: Props) {
...
+          {comments?.data && comments.data.length > 0 && (
+            <div className=" bg-gray-100 p-24 mt-16 rounded">
+              <ul className="flex flex-col gap-16">
+                {comments?.data.slice(0, 3).map((comment) => (
+                  <li>
+                    <p>
+                      <span className="font-bold text-gray-900">
+                        {nip19.npubEncode(comment.pubkey).slice(0, 10)}
+                      </span>{" "}
+                      <span className=" text-body5">
```
Comment (johnsBeharry) [inline src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx:116]:
```suggestion
                      <span className="text-body5 text-gray-400 italic">
```

File: src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx [hunk 2]
```diff
@@ -87,7 +92,49 @@ export default function StoryCard({ story }: Props) {
...
+                        {calcTimeSincePosting(
+                          new Date(comment.created_at * 1000).toISOString()
+                        )}
+                      </span>
+                    </p>
+                    <p className="text-gray-600 line-clamp-2">
```
Comment (johnsBeharry) [inline src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx:122]:
```suggestion
                    <p className="text-gray-600">
```

lets just show the whole comment.

Comment (MTG2000) [reply src/features/Posts/Components/PostCard/StoryCard/StoryCard.tsx:122]: image

Some comments could be very long...
See example above, and this is only 1 comment.

**Pull request merged**